### ISSUE TITLE
[stubgen] `TypeError: list or None object expected; got _frozen_importlib_external._NamespacePath` with pywin32

### ISSUE THREAD

**Issue by Avasam** [python/mypy#13774]
**Bug Report**

`TypeError: list or None object expected; got _frozen_importlib_external._NamespacePath` when running stubgen on most of pywin32's packages.

**To Reproduce**

```shell
# any of those
stubgen -p win32
stubgen -p win32comext
stubgen -p pythonwin
stubgen -p pywin32_system32
```

**Expected Behavior**

Generate stubs and not error out.

**Actual Behavior**

```py
Process Process-1:
Traceback (most recent call last):
  File "C:\Program Files\Python39\lib\multiprocessing\process.py", line 330, in _bootstrap
    self.run()
  File "C:\Program Files\Python39\lib\multiprocessing\process.py", line 117, in run
    self._target(*self._args, **self._kwargs)
  File "mypy\moduleinspect.py", line 95, in worker
  File "mypy\moduleinspect.py", line 54, in get_package_properties
TypeError: list or None object expected; got _frozen_importlib_external._NamespacePath
win32: Failed to import, skipping
```

**Your Environment**

- Mypy version used: 0.981 (compiled: yes)
- Mypy command-line flags: N/A
- Mypy configuration options from `mypy.ini` (and other config files): N/A
- Python version used: 3.9.13

**Comment by Avasam** [comment]
pywin32_system32 are just dlls anyway

After a bit of messing around, I think I got the proper syntax:

`stubgen -o stubs\pywin32\pythonwin -p win32ui`
`stubgen -o stubs\pywin32\pythonwin -p win32uiole`
OR (same result)
`stubgen -o stubs\pywin32 -p pythonwin.win32ui`
`stubgen -o stubs\pywin32 -p pythonwin.win32uiole`

Makes sense since you use it by doing `import win32ui` and `import win32uiole`

Similarly for win32comext and win32 I had to go through each package individually. Not sure if this is something stubgen can support automatically, or at least have a less cryptic error message.

**Comment by Avasam** [comment]
Closing as this is no longer replicatable. Likely got fixed since 0.981
